Kyrychenko/quizEntity/src/main/java/ui/QuizServlet.java

```diff
@@ -1,31 +1,72 @@
 package ui;
 
 import model.Quiz;
+import model.User;
 import service.QuizService;
+import service.UserService;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
 import java.io.IOException;
 import java.util.Arrays;
+import java.util.Objects;
 
 
 public class QuizServlet extends HttpServlet {
+    private static final String PARAM_NAME_USERNAME = "name";
```

Comment:
QuizServlet is not right name. If I see QuizServlet, I thought about Quiz logic. Not about login, registration and so on.